frontend/src/routes/AdminRoutes.js

```diff
@@ -49,6 +50,22 @@ function AdminRoutes() {
                 </ProtectedRoute>
               }
             />
+            <Route
+              path="tags/:id"
+              element={
+                <ProtectedRoute isAllowed={!!user && user.admin}>
+                  <TagForm />
+                </ProtectedRoute>
+              }
+            />
+            <Route
+              path="tags/create"
+              element={
+                <ProtectedRoute isAllowed={!!user && user.admin}>
+                  <TagForm />
```

Comment:
Since these are Forms, they should be embedded on a page and not used in the routes here.
A react `page` should be in the `pages` folder and this page should call a component (`TagForm`)

When making the page, try and keep the page design language similar to the rest of the admin pages.
`<div className="bg-white h-full overflow-y-scroll">` which can be found in Dashboard and UsersPage

Reply:
You guys should also add this in as a link on the admin side nav as well. Theres currently no way to get to these routes without typing in